index.html

```diff
@@ -523,7 +523,7 @@ <h3>Role Mapping Table</h3>
 				<tr id="role-map-caption">
 					<th><a class="role-reference" href="#caption"><code>caption</code></a></th>
 					<td class="role-msaa-ia2">
-						<span class="property">Role: <code>ROLE_SYSTEM_TEXT</code></span><br />
+						<span class="property">Role: <code>ROLE_SYSTEM_GROUPING</code></span><br />
```

Comment:
@aleventhal -- @jcsteh prefers this mapping in firefox: https://bugzilla.mozilla.org/show_bug.cgi?id=1809473

if you are ok with that I'll open an issue on chrome! (chrome exposes `ROLE_SYSTEM_TEXT`)

Reply:
I do too. The Chrome mapping makes no sense. ROLE_SYSTEM_TEXT is for text boxes.